packages/laravel-mailcoach-editor/src/Renderer/ButtonRenderer.php

```diff
@@ -18,7 +18,7 @@ public function render(): string
                                 <table border="0" cellpadding="0" cellspacing="0" role="presentation">
                                     <tr>
                                         <td>
-                                            <a href="{$this->data['url']}" class="button button-primary" target="_blank" rel="noopener">{$this->data['text']}</a>
+                                            <a href="{$this->data['link']}" class="button button-primary" target="_blank" rel="noopener">{$this->data['text']}</a>
```

Comment:
Won't this break old buttons? Could we do something like `$this->data['link'] ?? $this->data['url'] ?? ''` to be sure it won't throw any exceptions?

Reply:
I know we've disabled the button plugin so there won't be many people out there with buttons in their editor, but I'd rather be safe

Reply:
Actually, it might help when copy/paste. i will test it later to onfirm. 
anyway, i updated the file as you suggested.